src/psyclone/tests/psyir/frontend/fparser2_test.py

```diff
@@ -1183,42 +1204,6 @@ def test_process_not_supported_declarations():
     assert isinstance(l11sym.datatype, UnknownFortranType)
 
 
-def test_module_function_symbol(parser):
```

Comment:
It was removed because `generate_schedule` used to optionally create a Container too. Alternatively, the caller could pass in an existing Container. This test is exercising that functionality which no longer exists. However, I've extended a test in the subroutine_handler test file to do some of what was checked here.